webserver/pkgpkr/webservice/views.py

```diff
@@ -87,6 +88,14 @@ def repositories(request):
     one_or_more_with_dependencies = False
 
     for repo in repos:
+        # Updated Date
+        dateTime = repo['updatedAt']
+
+        # Convert time format e.g. 2020-03-16T13:03:34Z -> 2020-03-16 13:03:34
+        date = dateTime.split('T')[0]
+        time = dateTime.split('T')[-1][:-1]
+
+        repo['date'] = date + ' ' + time
```

Comment:
you can do (for consistency):

`repo['date'] = f'{date} {time}'`

Reply:
what do you think about moment.js that Daniel suggested above